api.py

```diff
@@ -368,6 +389,32 @@ def set_generic_data(dataset_id, key, data_string, **kwargs):
 
 
 def ingest_entities(dataset_id, data, **kwargs):
+    """
+    Upload an :ref:`entity set <conduce-entities>` to the Conduce datastore.
```

Comment:
Do we want to keep using the term entity set here and immediately below?

You're now referring to it as a list of entities elsewhere